src/lib/components/searchbox/main.svelte

```diff
@@ -173,7 +173,7 @@
 			class:border-neutral-200={showSuggestions}
 			class:dark:border-b-2={showSuggestions}
 			class:rounded-full={!showSuggestions}
-			class="size-full flex dark:bg-neutral-800 dark:shadow-none border dark:border-0 border-neutral-100 dark:border-neutral-700"
+			class="size-full flex dark:bg-neutral-800 dark:shadow-none border box-content dark:border-0 border-neutral-100 dark:border-neutral-700"
```

Comment:
Let's keep all of the border classes next to each other, I know it's a small nitpick but I've refactored the whole project around that so it's easier to understand all TailwindCSS styles when reading the code.

I would put this before size-full i.e. at the start like TW docs [here](https://tailwindcss.com/docs/box-sizing#excluding-borders-and-padding).

Reply:
I've put it there itentionally as it solves the border problem. That was my logic behind the placement as it's directly related to the borders in this instance. I'll reform it as you please though, it's no problem.